lib/checkuninitvar.cpp

```diff
@@ -998,7 +998,7 @@ bool CheckUninitVar::isVariableUsage(const Token *vartok, bool pointer, Alloc al
         const Token *parent = vartok->next()->astParent();
         while (Token::Match(parent, "[|."))
             parent = parent->astParent();
-        if (Token::simpleMatch(parent, "&") && !parent->astOperand2())
+        if (parent && Token::simpleMatch(parent, "&") && !parent->astOperand2())
```

Comment:
The match functions already check for null so the extra check at the beginning is redundant.